src/classes/BGE_BatchGiftEntry_UTIL.cls

```diff
@@ -102,4 +102,40 @@ public with sharing class BGE_BatchGiftEntry_UTIL {
         };
     }
 
+    /*******************************************************************************************************
+    * @description Returns the source field's options formatted for lightning:select if the source field is a picklist
```

Comment:
This is the tiniest remark, but can we prevent 'picklist' from wrapping to the next line?